python-backend/app/api/documents/expected/resources/expected_document_uploads.py

```diff
@@ -98,8 +99,17 @@ def put(self, expected_document_guid):
             db.session.commit()
         else:
             return self.create_error_payload(400, 'Must specify either Mine Document GIUD or Docuemnt Manager GUID'), 400
- 
-        return expected_document.json()           
+
+        try:
+            expected_document.exp_document_status_code = 'PRE'
```

Comment:
This endpoint is used by both minespace and CORE. This will set the status back to PRE whenever an inspector or proponent uploads a file, regardless of what the status was before.

Reply:
It was desired for only minespace to automatically set received date and status.

Reply:
So, you're saying that Minespace and CORE need to contain their own version of the business logic?

My impression was that any upload would need to be under review. If this isn't correct, I'll need a bit more clarity on how the described behaviour fits with the existing behaviour in CORE.